### DISCUSSION TITLE
Unclear instruction for quitting monitor

### DISCUSSION THREAD

**Discussion by urbite** [Mair/esp32-course#145] [General, feedback and testimonials]
In the **Installing and configuring VS Code** video, **at the 17:45 mark**, there is some instruction on how to exit monitor mode and get a terminal prompt again. I was unable to understand what keyboard combination instruction is being given.
[https://learnesp32.com/videos/1/3_installing%20and%20configuring%20vs%20code](https://learnesp32.com/videos/1/3_installing%20and%20configuring%20vs%20code)

It sounds to me like the instructor says _"...if I press CTRL + then 'that' key, then that terminations this window."_. After playing it back repeatedly I still can't understand what key should pressed in addition to CTRL - 'that' is what it sounds like to me.

After some googling, I found that CTRL+[ is the key combination to exit monitor mode and get the command prompt back.
[https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-monitor.html](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-monitor.html)

A minor point, but it added to my confusion was the statement that _"...then that terminates this window"_. But no window is terminated or closed, only the monitor mode is terminated and terminal mode is active again - in the same window.

A suggestion for rewording: _"...if I press CTRL + right-square-bracket, this terminates monitor mode and returns control to the terminal command prompt."_

Excellent course and videos thus far. Not trying to be picky, just a suggestion that might make this step easier to follow for others.

**Comment by noweare1** [comment]
ctrl key + ]    (closed square bracket) at same time
